pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -432,12 +449,15 @@ func (nc *Controller) doNoScheduleTaintingPass(node *v1.Node) error {
 	// Map node's condition to Taints.
 	taints := []v1.Taint{}
 	for _, condition := range node.Status.Conditions {
-		if _, found := nodeConditionToTaintKeyMap[condition.Type]; found {
-			if condition.Status == v1.ConditionTrue {
+		if condition.Status == v1.ConditionTrue {
```

Comment:
check true for memory pressure, out of disk and so on. check false for ready.

Reply:
Ok, I will check it.